lib/operations_api.js

```diff
@@ -93,12 +96,13 @@ function OperationsApi(gaxGrpc, grpcClients, opts) {
     'gax/' + gax.version,
     'nodejs/' + process.version].join(' ');
 
+  var bundleDescriptors = {
```

Comment:
This is now added to createApiCall() below -- but I've changed the code generator, so this is gone now.